redis_views/response.py

```diff
@@ -19,6 +31,10 @@ def resolve_template(self, template):
 
         template_body = self.connection.get(template)
         if not template_body:
+            # log an error message if the template key was not found in the DB
+            msg = 'Reids Template: Could not find the template key {}'.format(template)
```

Comment:
Typo: "Redis". I think it might also not be necessary to say `Redis Template` because I think it would be namespaced using the logger above.